src/components/MyActivities/pages/Loggedin/MyActivitiesHelp.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import '../../MyActivitiesStyle.scss';
+import help from './help';
+
+function MyActivitiesHelp() {
+  const [helpId, setHelpId] = useState(0);
+
+  return (
+    <>
+      {localStorage.getItem('help-visited') !== 'true' ? (
+        <>
+          <div
+            className={
+              helpId == 0
```

Comment:
As I mentioned in help.json file id is not clear here and a key or string id would be better.